speakeasy/windows/win32.py

```diff
@@ -269,8 +269,23 @@ def run_module(self, module, all_entrypoints=False):
 
                     run.type = 'export.%s' % (fn)
                     run.start_addr = exp.address
-                    # Here we set dummy args to pass into the export function
-                    run.args = args
+                    if exp.name == 'ServiceMain':
+                        # argc: 1
+                        # argv:
+                        #   0x00: (argv[0]) pointer to +0x10 -+
+                        #   0x08: (argv[1]) 0x0               |
+                        #   0x10: "IPRIP"  <------------------+
+                        svc_name = "IPRIP\x00".encode('utf-16le')
```

Comment:
we don't really know if we're in A or W mode. if we use W mode here, and we're actually in A mode, then subsequent str calls will think the service name is `I` which isn't perfect but maybe acceptable ;-)